src/store/state.js

```diff
@@ -0,0 +1,5 @@
+const state = {
```

Comment:
For all your files, we should ideally have an end line. Please configure this in your text editor and then you won't have to think about it next time.

Reply:
Yup. Added on my viscose